nifi-nar-bundles/nifi-jms-bundle/nifi-jms-processors/src/main/java/org/apache/nifi/jms/processors/AbstractJMSProcessor.java

```diff
@@ -105,6 +112,7 @@
         propertyDescriptors.add(CF_SERVICE);
         propertyDescriptors.add(DESTINATION);
         propertyDescriptors.add(DESTINATION_TYPE);
+        propertyDescriptors.add(SUBSCRIPTION_NAME);
```

Comment:
Adding `Subscription Name` at AbstractJMSProcessor makes both ConsumeJMS and PublishJMS to have the property. I think we want only ConsumeJMS to have the property. Should it be done at ConsumeJMS?